utils/wasm-gen/src/generator/syscalls/imports.rs

```diff
@@ -695,6 +696,93 @@ impl<'a, 'b> SysCallsImportsGenerator<'a, 'b> {
         Ok(call_indexes_handle)
     }
 
+    fn generate_reply_deposit(
+        &mut self,
+        syscall: SysCallName,
+    ) -> Result<CallIndexesHandle, PreciseSysCallError> {
+        let [send_input_idx, reply_deposit_idx] =
+            self.invocable_syscalls_indexes(InvocableSysCall::required_imports(syscall))?;
+
+        let mid_ptr = self.reserve_memory();
+
+        let precise_reply_deposit_invocation = [
+            // Pointer to pid_value argument of HashWithValue type.
+            Instruction::GetLocal(0),
+            // Pointer to offset defining starting index in the received message payload.
+            Instruction::GetLocal(1),
+            // Pointer to length of the slice of the received message payload.
```

Comment:
```suggestion
            // Length of the slice of the received message payload.
```
As I understood from signature

Reply:
yup

Reply:
done